price-feeder/price-feeder.example.toml

```diff
@@ -211,21 +207,6 @@ providers = [
 ]
 quote = "USD"
 
-[[currency_pairs]]
-base = "BTC"
-providers = [
-  "coinbase",
-]
-quote = "USD"
-
-[[currency_pairs]]
-base = "BTC"
-providers = [
-  "huobi",
-  "kraken",
-]
-quote = "USDT"
```

Comment:
why removing these important pair from kraken and coinbase?

Reply:
Our accept list in the oracle is tracking the price of  `wbtc` not `btc`, so I don't see where the price data of `btc` would be getting used atm.

Reply:
Could you add in changelog that we are removing BTC from the template, and explain why. Also add a note to the RELEASE_NOTES file in price-feeder.